src/backend/distributed/master/master_metadata_utility.c

```diff
@@ -765,6 +767,50 @@ TableOwner(Oid relationId)
 
 
 /*
+ * SchemaOwner returns the name of the owner of the specified schema.
+ */
+char *
+SchemaOwner(Oid schemaId)
```

Comment:
Seems that we don't have a nice file for this function right now. Maybe just keep it in master_node_protocol where it is called and make it static.